src/parser/name_resolution.c

```diff
@@ -1114,11 +1115,40 @@ pt_set_is_view_spec (PARSER_CONTEXT * parser, PT_NODE * node, void *arg, int *co
       /* Reset query id # */
       node->info.query.id = (UINTPTR) node;
       node->info.query.is_view_spec = 1;
+
+      if (do_not_replace_orderby)
+	{
+	  node->flag.do_not_replace_orderby = 1;
+	}
     }
 
   return node;
 }				/* pt_set_is_view_spec */
 
+/*
+ * pt_set_do_not_replace_orderby () -
+ *   return:
+ *   parser(in):
+ *   node(in):
+ *   arg(in):
+ *   continue_walk(in):
+ */
+PT_NODE *
+pt_set_do_not_replace_orderby (PARSER_CONTEXT * parser, PT_NODE * node, void *arg, int *continue_walk)
+{
+  if (!node)
+    {
+      return node;
+    }
+
+  if (pt_is_query (node))
+    {
+      node->flag.do_not_replace_orderby = 1;
+    }
```

Comment:
`pt_is_query()` checks node is NULL.

```suggestion
  if (pt_is_query (node))
    {
      node->flag.do_not_replace_orderby = 1;
    }
```

Reply:
Okay. thanks.